### PR TITLE
Adding codeowner

### PR DESCRIPTION
**Pull request by Kathar-Patcha-abdul-rahim**

### CODE REVIEW

**Comment by Lilall5829** [comment]
asdasd

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Code owner for all files in the repository
+* @Kathar-Patcha-abdul-rahim
```
Comment (Lilall5829) [inline .github/CODEOWNERS:2]: asdasd

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Code owner for all files in the repository
```
Comment (Lilall5829) [inline .github/CODEOWNERS:1]: asdasd

**Review by Lilall5829 — APPROVED** [review]

**Review by marufahmed-hub — COMMENTED** [review]
Reviewed

**Review by marufahmed-hub — APPROVED** [review]

**Review by marufahmed-hub — APPROVED** [review]

**Pull request merged**